src/main/java/main/internet/shop/storage/Storage.java

```diff
@@ -0,0 +1,68 @@
+package main.internet.shop.storage;
+
+import main.internet.shop.models.Order;
+import main.internet.shop.models.Product;
+import main.internet.shop.models.ShoppingCart;
+import main.internet.shop.models.User;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+
+    private static final Map<Long, Order> orderStorage = new HashMap<>();
```

Comment:
each of our model has it's field id, so let's use List to store them

Reply:
Why list? isn't it faster to search in map by field id as key?

Reply:
it's information duplication. You are using orderId as key of map where we store orders(each of them also have this id)